.github/CODEOWNERS

```diff
@@ -0,0 +1,192 @@
+# This file lists the owners of code in different areas of the lnd codebase
+# Codeowners will own the review for the changes being merged into their
+# respective areas
+
+# aezeed
+/aezeed/ @yyforyongyu
+
+# alias manager
+/aliasmgr/ @yyforyongyu
+
+# amp
+/amp/ @yyforyongyu
+
+# auto pilot
+/autopilot/ @ellemouton @bhandras
+
+# batch
+/batch/ @bhandras
+
+# block cache
+/blockcache/ @yyforyongyu
+
+# brontide
+/brontide/ @Roasbeef @yyforyongyu
+
+# buffer
+/buffer/ @yyforyongyu
+
+# build
+/build/ @Roasbeef @guggero
+
+# certificates
+/cert/ @guggero
+
+# chain notifications
+/chainntnfs/ @Roasbeef @yyforyongyu
+
+# chain registry
+/chainreg/ @ellemouton
+
+# channel acceptor
+/chanacceptor/ @Crypt-iQ
+
+# channel backup
+/chanbackup/ @guggero
+
+# channel fitness
+/chanfitness/ @yyforyongyu
+
+# channel db
+/channeldb/ @Roasbeef @yyforyongyu
+
+# channel notifier
+/channelnotifier/ @yyforyongyu
+
+# clock
+/clock/ @bhandras
+
+# cluster
+/cluster/ @bhandras
+
+# command line
+/cmd/ @ellemouton
+
+# contract court
+/contractcourt/ @yyforyongyu
+
+# contrib
+/contrib/ @guggero
+
+# discovery
+/discovery/ @Roasbeef
```

Comment:
can I be added as a secondary here & to `routing` too? im getting more experience in those now with all the gossip work